style.css

```diff
@@ -0,0 +1,463 @@
+  z-index: 10;
+  padding-left: 26px;
+  margin-top: -165px;
+  font-weight: bold;
+  line-height: 51px;
+  font-size: 50px;
+}
+
+.up-section__description,
+.up-section__contacts {
+  display: none;
+}
+
+.up-section__images {
+  z-index: -10;
+  height: 529px;
+  overflow: hidden;
+}
+
+.up-section__collection-image img {
+  width: 100%;
+  height: 100%;
+  -o-object-fit: cover;
+  object-fit: cover;
+  -o-object-position: 0 -55px;
+  object-position: 0 -55px;
+}
+
+.up-section__desription {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 9px;
+  line-height: 13px;
+  color: #000;
+}
+
+.up-section__scrooll {
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: -40px;
+  width: 27px;
+  height: 46px;
+  margin: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  cursor: pointer;
+}
+
+@media (min-width: 992px) {
+  .header {
+    min-height: 92px;
+  }
+
+  .header__head {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    min-width: 461px;
+    background: #000;
+    padding: 0;
+    margin: 0;
+  }
+
+  .header__logo {
+    margin-top: 42px;
+    margin-left: 55px;
+  }
+
+  .header__lang {
+    position: relative;
+    display: block;
+    width: 50px;
+    -ms-flex-item-align: center;
+    -ms-grid-row-align: center;
+    align-self: center;
+    margin-right: 22px;
+    margin-top: 20px;
+  }
+
+  .header__lang-option {
+    position: absolute;
+    width: 31px;
+    height: 25px;
+    left: 388px;
+    top: 0;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    text-align: right;
+    text-decoration: none;
+    color: #fff;
+  }
+
+  .header__lang-option:hover {
+    border-bottom: solid 2px #eeddd0;
+  }
+
+  .header__lang-option::after {
+    position: absolute;
+    top: 3px;
+    left: 45px;
+    display: block;
+    content: '';
+    width: 7px;
+    height: 6px;
+    border: 2px solid #fff;
+    border-right-color: transparent;
+    border-top-color: transparent;
+    -webkit-transform: rotate(-44deg);
+    -ms-transform: rotate(-44deg);
+    transform: rotate(-44deg);
+  }
+
+  .header__lang-option:hover::after {
+    transform: rotate(135deg);
+  }
+
+  .header__menu {
+    width: 92px;
+    height: 92px;
+  }
+
+  .header__menu:hover {
+    background-color: #eeddd0;
+  }
+
+  .header__burger-line:first-of-type {
+    margin-bottom: 5px;
+  }
+
+  .header__burger-line {
+    width: 35px;
+    height: 4px;
+  }
+
+  .up-section__desription {
+    line-height: 21px;
+    font-size: 15px;
+    font-weight: normal;
+  }
+
+  .up-section__collection-description{
+    width: 475px;
+    margin-left: 45px;
+  }
+
+  .up-section__title {
+    padding-left: 10px;
+    margin-top: 108px;
+    margin-bottom: 2px;
+    width: 475px;
+    height: 164px;
+    font-weight: bold;
+    line-height: 82px;
+    font-size: 80px;
+    }
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5676737)